x-pack/test/security_solution_cypress/cypress/tasks/api_calls/elasticsearch.ts

```diff
@@ -9,8 +9,12 @@ import { rootRequest } from '../common';
 export const deleteIndex = (index: string) => {
   rootRequest({
     method: 'DELETE',
-    url: `${Cypress.env('ELASTICSEARCH_URL')}/${index}?refresh=wait_for`,
-    headers: { 'kbn-xsrf': 'cypress-creds', 'x-elastic-internal-origin': 'security-solution' },
+    url: `${Cypress.env('ELASTICSEARCH_URL')}/${index}`,
```

Comment:
Why we're removing `refresh=wait_for`?

Reply:
`refresh` is an unsupported param for index deletion API. This request is executed with `failOnStatusCode` set to  `false` so we weren't able to catch the error but the burnout test failed. Investigation revealed the problem.

Reply:
Awesome, thank you for fixing this @maximpn!